CONTRIBUTING.md

```diff
@@ -502,6 +510,45 @@ kubectl -n crossplane-system scale deploy crossplane --replicas=0
 make run
 ```
 
+## Code Review Process
+
+All Pull Requests (PR), whether written by a Crossplane maintainer or a
+community member, must go through code review. This ensures that the code is
+correct, maintainable, and secure.
+
+We encourage anyone in the community to conduct a code review on a PR,
+however, we require the following approvals before merging a PR:
+- At least one approval from [Reviewers] or [Code Owners]
+- At least one approval from [Maintainers]
+
+When opening a PR, GitHub will automatically assign the
+[crossplane-reviewers] and [crossplane-maintainers] teams, as well as
+any [Code Owners]. The PR author may also add any additional reviewers they
+might want to have a look at the PR.
+
+If the PR author has a specific code reviewer in mind, we encourage them to
+remove the auto-assigned reviewers and directly add the individuals they would
+like to review the code.
```

Comment:
I'm a little on the fence about this guidance.

* I don't think most folks (i.e. folks who aren't reviewers or maintainers) are able to assign reviewers.
* I'm wary of encouraging the average community member to pick and choose their reviewers; I would not be surprised if this resulted in a lot of PRs being assigned to the folks the community members perceive to be the most responsive and thus not spreading the load fairly.

Perhaps you could clarify that only Crossplane organization members have this power? Or I could be easily convinced to just leave it out and have it be "tribal knowledge" amongst reviewers.

Reply:
Going to remove based on the clarified understanding of the assignment rules.